engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -1204,10 +1204,12 @@ namespace dmGameSystem
                     total_count += world->m_VertexBufferVertexCounts[batch_index];
                 }
 
-                world->m_DispatchCount++;
-
-                DM_PROPERTY_ADD_U32(rmtp_ModelVertexCount, total_count);
-                DM_PROPERTY_ADD_U32(rmtp_ModelVertexSize, total_count * sizeof(dmRig::RigModelVertex));
+                if (total_count > 0)
```

Comment:
Probably not for this PR, but I thought I should bring it up:

I noticed that the "batch index" is predetermined by the `m_MinorOrder`in the CompModelRender(), but is that really the best way to do it?

Shouldn't the vertex buffer index be more dynamic, and be counted upwards, as the `RENDER_LIST_OPERATION_BATCH` calls are coming in?
It feels a bit unintuitive 🤔

Reply:
We would have to deal with that based on local vs world space, local spaced models uses the "baked" vertex buffers already afaik. But yes, it would be great if we simplified this a bit